eng/emitter-package-lock.json

```diff
@@ -9,7 +9,7 @@
         "@azure-tools/typespec-autorest": "0.35.0",
         "@azure-tools/typespec-azure-core": "0.35.0",
         "@azure-tools/typespec-client-generator-core": "0.35.0",
-        "@azure-tools/typespec-ts": "0.18.0",
+        "@azure-tools/typespec-ts": "0.19.0",
```

Comment:
@minhanh-phan The emitter-package-lock.json is the pair file with emitter-package.json file which would be used in automation ci to lock the emitter versions. 

You could follow the steps to prepare this.